src/components/Explore/utils.ts

```diff
@@ -151,10 +159,13 @@ export const EXPLORE_CHART_IDS = Object.values(exploreMetricData).map(
   metric => metric.chartId,
 );
 
-const hasValue = (point: Column) => isNumber(point.y);
+const missingValue = (point: Column) => !isNumber(point.y);
 
+/**
+ * Remove points without y values at the begining and end of the series
```

Comment:
Perhaps add a TODO about the fact that we still should deal with missing y values in the middle of the series. 🤷